commands/render.js

```diff
@@ -143,31 +135,29 @@ function renderFrames(records, options) {
       }
 
       progressBar.tick();
-      if (progressBar.complete) {
-        console.log('Finished rendering frames in ' + (Date.now() - start) + 'ms.');
-        resolve();
-      }
     }
 
-    // React on rendering errors observed on its stderr
+    // Track rendering errors observed on stderr
     function onError(error) {
 
-      // First kill the rendering process
-      render.kill();
-
-      // Check for GPU error
-      if (!!error && error instanceof Buffer && error.toString('utf8').includes('GL_INVALID_OPERATION : glBufferData')) {
-        console.log('Rendering failed due to Electron error \'GL_INVALID_OPERATION: glBufferData\'!'); 
-        if (!attemptedRenderWithoutGPU) {
-          console.log('Retrying rendering with disabled GPU...');
-          attemptedRenderWithoutGPU = true;
-          return spawnRender('--disable-gpu');
-        }
+      // If error is Buffer, print it, otherwise reject
+      if (!!error && error instanceof Buffer) {
+        console.log(di.chalk.yellow(error.toString('utf8')));
```

Comment:
Add a hint to identify the error source
```suggestion
        console.log(di.chalk.yellow(`[render] ${error.toString('utf8')}`));
```